src/client/graphics/SpriteLoader.ts

```diff
@@ -5,28 +5,35 @@ import mirvSprite from "../../../resources/sprites/mirv2.png";
 import samMissileSprite from "../../../resources/sprites/samMissile.png";
 import tradeShipSprite from "../../../resources/sprites/tradeship.png";
 import trainCarriageSprite from "../../../resources/sprites/trainCarriage.png";
-import trainCarriageLoadedSprite from "../../../resources/sprites/trainCarriageLoaded.png";
+import trainLoadedCarriageSprite from "../../../resources/sprites/trainCarriageLoaded.png";
 import trainEngineSprite from "../../../resources/sprites/trainEngine.png";
 import transportShipSprite from "../../../resources/sprites/transportship.png";
 import warshipSprite from "../../../resources/sprites/warship.png";
 import { Theme } from "../../core/configuration/Config";
-import { UnitType } from "../../core/game/Game";
+import { TrainType, UnitType } from "../../core/game/Game";
 import { UnitView } from "../../core/game/GameView";
 
-const SPRITE_CONFIG: Partial<Record<UnitType, string>> = {
+// Can't reuse TrainType because loaded is not a type, just an attribute
+enum TrainTypeSprite {
+  Engine = "Engine",
+  Carriage = "Carriage",
+  LoadedCarriage = "LoadedCarriage",
+}
```

Comment:
We're trying to move away from enums (see #966).
```suggestion
type TrainTypeSprite =
  | "Engine",
  | "Carriage",
  | "LoadedCarriage";
```